torch/distributed/tensor/placement_types.py

```diff
@@ -718,11 +661,6 @@ def __str__(self) -> str:
         """
         return "P"
```

Comment:
This is a super weird str method btw

Reply:
I agree but it's definitely out of scope